src/Commands/RunJobs.php

```diff
@@ -0,0 +1,144 @@
+
+        // ---------------------------------------------------
+        //  Prepare dispatcher and success and error logs
+        // ---------------------------------------------------
+
+        $dispatcher = $this->getDispatcher($input);
+
+        $jobs_ran = $jobs_failed = [];
+
+        $dispatcher->getQueue()->onJobFailure(function (Job $job) use (&$jobs_failed) {
+            $job_id = $job->getQueueId();
+
+            if (!in_array($job_id, $jobs_failed)) {
+                $jobs_failed[] = $job_id;
+            }
+        });
+
+        if ($jobs_count = $dispatcher->getQueue()->count()) {
+
+            // ---------------------------------------------------
+            //  Set max execution time for the jobs in queue
+            // ---------------------------------------------------
+
+            $max_execution_time = (integer)$input->getOption('seconds');
+
+            $output->writeln("There are " . $dispatcher->getQueue()->count() . " jobs in the queue. Preparing to work for {$max_execution_time} seconds.");
+
+            $work_until = time() + $max_execution_time; // Assume that we spent 1 second bootstrapping the command
+            // ---------------------------------------------------
+            //  Set channels for the jobs in queue
+            // ---------------------------------------------------
+            $channels_string = $input->getOption('channels');
+            $channels = explode(",",$channels_string);
+            if(empty($channels)){
+                $this->abort('Channel argument is expected, pleas try again.',22,$input,$output);//ERROR_BAD_COMMAND
+            }
+            // ---------------------------------------------------
+            //  Enter the execution loop
+            // ---------------------------------------------------
+
+            do {
+                if ($next_in_line = call_user_func_array(array($dispatcher->getQueue(),'nextInLine'),$channels)) {
+                    $log->info('Running job #' . $next_in_line->getQueueId() . ' (' . get_class($next_in_line) . ')');
+
+                    if ($output->getVerbosity()) {
+                        $output->writeln('<info>OK</info> Running job #' . $next_in_line->getQueueId() . ' for instance #' . $next_in_line->getData()['instance_id'] . ' (' . get_class($next_in_line) . ')');
+                    }
+
+                    $dispatcher->getQueue()->execute($next_in_line);
+
+                    if ($output->getVerbosity()) {
+                        $output->writeln('<info>OK</info> Job #' . $next_in_line->getQueueId() . ' done');
+                    }
+
+                    $job_id = $next_in_line->getQueueId();
+
+                    if (!in_array($job_id, $jobs_ran)) {
+                        $jobs_ran[] = $job_id;
+                    }
+                } else {
+                    if ($dispatcher->getQueue()->count()) {
+                        $log->info('Reservation collision');
```

Comment:
Change the message. It should be `Next in line not found, or we have a reservation collision`